Reply/app/src/main/java/com/materialstudies/reply/ui/MainActivity.kt

```diff
@@ -196,4 +249,81 @@ class MainActivity : AppCompatActivity(),
             }
         }
     }
+
+    private fun adaptToBottomNavScreen(
```

Comment:
Okay, I originally had this but with the addition of the XLarge screen it was a little confusing because the Medium / Large screens have the same implementation for the Activity Level. But I can change it to `adaptToMediumAndLarge`.